index.html

```diff
@@ -5,27 +5,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Campos e Requisitos da Tela de Votação</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.2/css/bulma.min.css" media="print" onload="this.media='all'">
 </head>
 
 <body>
-    
-    <h1>O que você acha do novo design?</h1>
 
-    <form action="" method="Post">
-        <div>
-            <input type="radio" name="voto" value="Ótimo!" >
-            <label for="">Ótimo!</label>
-        </div>
-        <div>
-            <input type="radio" name="voto" value="Bom, mas pode melhorar">
-            <label for=""> Bom, mas pode melhorar</label>
-        </div>
-        <div>
-            <input type="radio" name="voto" value="Não gostei">
-            <label for="">Não gostei</label>
+    <section class="section">
+        <div class="columns is-centered">
+            <div class="column is-half box">
+                <h1 class="title has-text-centered has-text-primary is-size-2 has-text-weight-bold">
+                    O que você acha do novo design?
+                </h1>
+
+                <form action="" method="post">
+                    <fieldset>
+                        
+                        <div class="columns is-centered mt-4">
+                            <div class="control">
+                                <input type="radio" id="otimo" name="voto" value="Ótimo!" required>
+                                <label for="otimo">Ótimo!</label>
+                                <br>
+
+                                <input type="radio" id="bom" name="voto" value="Bom, mas pode melhorar">
+                                <label for="bom">Bom, mas pode melhorar</label>
+                                <br>
+
+                                <input type="radio" id="nao-gostei" name="voto" value="Não gostei">
+                                <label for="nao-gostei">Não gostei</label>
+                            </div>
+                        </div>
+                    </fieldset>
+
+                    <div class="columns is-centered mt-4">
```

Comment:
Como todos os elementos vão ser alinhados pela classe pai através do flexbox, pode remover o `columns` e o `is-centered`.

Reply:
check